### ISSUE TITLE
SQL syntax on Match predicate

### ISSUE THREAD

**Issue by freshcoat** [ripaclub/sphinxsearch#47]
Hello,

When using the Match predicate I am receiving an SQL syntax error.
`Statement could not be executed (42000 - 1064 - You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''keyword') AND online = 1 LIMIT 0,20' at line 1)`

Example code:

```
$rowset = $search->search('users', function(Select $select) {
$select ->columns(array('name'))
        ->where(new Match('?', 'keyword'))
        ->where(array('online = ?' => 1))
        ->limit(20);
});
```

When running the same code without the 'Match' predicate, codes runs correct and returns a set with results from te database.

**Comment by leogr** [comment]
Hi @klasver,
Have you solved your problem? 
If not yet, please let us know which version of Sphinx you are using.

**Comment by freshcoat** [comment]
Hello,

First of all, I am sorry I dropped this issue probably too fast. I noticed the connection to SphinxQL wasn't set right and didn't working. Match predicate wasn't working because of this issue.

I wasn't able to track the error easily till I implemented more Exception catchers in Search.php to determine the connection was created successfully or failed which failed in my case.

Thanks for your quick response and the good work you guys do.

Regards,
Nicky
